notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1213 @@
+{
```

Comment:
<p>Suggest moving this into the 2 sections that reference it: Create a bucket and dataset. Otherwise, seems misplaced to stand on it own and discussed in terms of appending it to other resources.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>